gems/activesupport/6.0/activesupport.rbs

```diff
@@ -501,7 +501,7 @@ class Array[unchecked out Elem]
   #
   # The differences with <tt>Kernel#Array</tt> explained above
   # apply to the rest of <tt>object</tt>s.
-  def self.wrap: (nil) -> []
+  def self.wrap: (nil) -> Array[untyped]
                | [T] (_ToAry[T] array_like) -> Array[T]
                | [T] (T ele) -> [T]
```

Comment:
I have another idea to avoid the problem. It is reordering the definition not to match `untyped` to `(nil) -> []`.

```suggestion
  def self.wrap: [T] (_ToAry[T] array_like) -> Array[T]
               | (nil) -> Array[untyped]
               | [T] (T ele) -> [T]
```

This will also resolve my case.  But it will still not go well in the case of nil.

Reply:
Are you saying your original solution in this PR does not resolve your issue?

I think your original suggestion is fine although I haven't dug too deep into all of its use cases. It is unfortunate that there are no overloaded method definitions for the empty array `[]`.

I did confirm that `Array(nil).map(&:to_sym)` also blows up.

Reply:
Oh, sorry for confusing you. My first proposal resolves my problem. So please forget this approach.